posts/first-lean-release.md

```diff
@@ -0,0 +1,49 @@
+---
+author: 'Scott Morrison'
+category: 'announcement'
+date: 2023-08-29 16:30:00 UTC+02:00
+description: 'The first official release of Lean 4, and our plan for regular releases.'
+has_math: true
+link: ''
+slug: first-lean-release
+tags: ''
+title: The first official release of Lean 4
+type: text
+---
+
+Lean 4 has just made its first official stable release, with the arrival of [`v4.0.0`](https://github.com/leanprover/lean4/releases/tag/v4.0.0).
+We're excited to begin the transition from only providing nightly releases towards having regular stable releases.
+
+<!-- TEASER_END -->
+## Why?
+
+Currently most of the Lean ecosystem works with nightly releases,
+and projects depending on branches (often `master` or `main`!),
+rather than fixed versions of upstream dependencies.
+
+As Lean matures into a significant programming language,
+as well as an even better theorem prover for modern mathematics and software/hardware verification,
+the ecosystem will grow.
+We want to make sure that everyone can find compatible versions of libraries,
+and that everything **just works**.
+
+It's a long road to get there,
+but the Lean Focused Research Organization intends to dedicate resources
```

Comment:
A link to leanFRO's website would be good here

Reply:
Done!